interpreter/cling/lib/Interpreter/IncrementalExecutor.cpp

```diff
@@ -91,6 +91,19 @@ std::unique_ptr<TargetMachine>
   CodeModel::Model CMModel = CodeModel::JITDefault;
   CodeGenOpt::Level OptLevel = CodeGenOpt::Less;
 
+  const char* OptLevelEnvConfig = std::getenv("ROOT_OPTIMIZATION_LEVEL");
```

Comment:
Implemented here : https://github.com/Y--/root/commit/97a6d48dc1cd40ab414d76cd6bbfe48343ba144d and there https://github.com/Y--/root/commit/5f8c40035a4f9305a53384a35419d3f4d508021b